schedule/ha/bv/basic_cluster_node.yaml

```diff
@@ -40,7 +40,7 @@ vars:
   # LVM locking daemon used since SLE 15
   USE_LVMLOCKD: '1'
   USE_SUPPORT_SERVER: '1'
-  VIRTIO_CONSOLE: '0'
+  VIRTIO_CONSOLE: '1'
```

Comment:
Why this change is needed?

Reply:
The `VIRTIO_CONSOLE: '0'` here will overwrite the `VIRTIO_CONSOLE: '1'` in job template yaml file `settings`